### PR TITLE
Feature/omk 11062&10608

### PR DESCRIPTION
**Pull request by whatgwan**
check avx support and added a last daemon start check

### CODE REVIEW

File: admin/support.pl [hunk 1]
```diff
@@ -330,6 +342,27 @@ sub collect_evidence
 		cp("/etc/systemd/system/nmis9d.service", "$targetdir/system_status/init");
 	}
 
+	my @daemons = ("nmis9d", "apache2", "httpd", "mongod"); # Add or remove daemons as needed
+
+	open(my $daemon_info_fh, '>>', "$targetdir/system_status/init/daemon_status") or die "Could not open file '$targetdir/system_status/init/daemon_status' $!";
+
+        foreach my $daemon (@daemons) {
+                my $status = `systemctl is-active $daemon 2>/dev/null`;
```
Comment (markdueck1) [inline admin/support.pl]: do we need to see if systemctl is available on the box?

File: admin/support.pl [hunk 2]
```diff
@@ -330,6 +342,27 @@ sub collect_evidence
...
+                chomp $status;
```
Comment (markdueck1) [inline admin/support.pl]: this should check to see if $status is something before it starts working with it

**Review by markdueck1 — CHANGES_REQUESTED** [review]

File: admin/support.pl (hunk 1 above)
Comment (whatgwan) [reply admin/support.pl]: I have added in your suggestion

File: admin/support.pl (hunk 2 above)
Comment (whatgwan) [reply admin/support.pl]: I have added in your suggestion

**Review by markdueck1 — APPROVED** [review]

**Pull request merged**